packages/gdl-frontend/package.json

```diff
@@ -23,8 +23,8 @@
     "@lingui/react": "2.7.2",
     "@material-ui/core": "3.6.1",
     "@material-ui/icons": "4.4.1",
-    "@sentry/browser": "4.4.2",
-    "@sentry/node": "4.4.2",
+    "@sentry/browser": "4.5.4",
+    "@sentry/node": "4.5.4",
```

Comment:
Var det en spesiell grunn til at det ble akkurat denne versjonen?

Reply:
Ville ikkje ta for masse i for ikkje innføre breaking changes men likevel fikse en feil som fører til mange hendelser i sentrykonsollet.